test.disabled/ejabberd_tests/tests/s2s_helper.erl

```diff
@@ -0,0 +1,136 @@
+-module(s2s_helper).
+-export([suite/1]).
+-export([init_s2s/2]).
+-export([end_s2s/1]).
+-export([configure_s2s/2]).
+
+-import(distributed_helper, [rpc/4, rpc/5]).
+
+-include_lib("escalus/include/escalus.hrl").
+-include_lib("common_test/include/ct.hrl").
+-record(s2s_opts, {
+          node1_s2s_certfile = undefined,
+          node1_s2s_use_starttls = undefined,
+          node2_s2s_certfile = undefined,
+          node2_s2s_use_starttls = undefined
+         }).
+
+
+suite(Config) ->
+    require_s2s_nodes() ++ Config.
+
+require_s2s_nodes() ->
+    [{require, mim_node, {hosts, mim, node}},
+     {require, fed_node, {hosts, fed, node}}].
+
+init_s2s(Config, CoverEnabled) when is_boolean(CoverEnabled) ->
```

Comment:
Enabling cover twice during build process would cause strange errors on travis.
So, cover is enabled just for s2s_SUITE